smtp/main.go

```diff
@@ -137,13 +138,20 @@ func (s *smtpNotifier) SendNotification(ctx context.Context, build *cbpb.Build)
 		log.V(2).Infof("no mail for event:\n%s", proto.MarshalTextString(build))
```

Comment:
Is there a unit test for this method? It'd be nice to see a unit test for the Resolve function